Detections/MultipleDataSources/IridiumIOCs.yaml

```diff
@@ -0,0 +1,121 @@
+  - connectorId: AzureMonitor(WireData)
+    dataTypes:
+      - WireData
+  - connectorId: AzureMonitor(IIS)
+    dataTypes:
+      - W3CIISLog
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+query: | 
+ let timeframe = 1d;
+ let IPList = dynamic(["154.223.45.38","185.141.207.140","185.234.73.19","216.245.210.106","51.91.48.210","46.255.230.229"]);
+ (union isfuzzy=true
+ (CommonSecurityLog
+ | where TimeGenerated >= ago(timeframe) 
+ | where isnotempty(SourceIP) or isnotempty(DestinationIP)
+ | where SourceIP in (IPList) or DestinationIP in (IPList) or Message has_any (IPList)
+ | extend IPMatch = case(SourceIP in (IPList), "SourceIP", DestinationIP in (IPList), "DestinationIP", "Message") 
+ | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated) by SourceIP, DestinationIP, DeviceProduct, DeviceAction, Message, Protocol, SourcePort, DestinationPort, DeviceAddress, DeviceName, IPMatch
+ | extend timestamp = StartTimeUtc, IPCustomEntity = case(IPMatch == "SourceIP", SourceIP, IPMatch == "DestinationIP", DestinationIP, "IP in Message Field") 
+ ),
+ (OfficeActivity
+ | where TimeGenerated >= ago(timeframe) 
+ |extend SourceIPAddress = ClientIP, Account = UserId
+ | where  SourceIPAddress in (IPList)
+ | extend timestamp = TimeGenerated , IPCustomEntity = SourceIPAddress , AccountCustomEntity = Account
+ ),
+ (DnsEvents 
+ | where TimeGenerated >= ago(timeframe) 
+ | extend DestinationIPAddress = IPAddresses,  Host = Computer
+ | where  DestinationIPAddress has_any (IPList) 
+ | extend timestamp = TimeGenerated, IPCustomEntity = DestinationIPAddress, HostCustomEntity = Host
+ ),
+ (VMConnection 
+ | where TimeGenerated >= ago(timeframe) 
+ | where isnotempty(SourceIp) or isnotempty(DestinationIp) 
+ | where SourceIp in (IPList) or DestinationIp in (IPList) 
+ | extend IPMatch = case( SourceIp in (IPList), "SourceIP", DestinationIp in (IPList), "DestinationIP", "None") 
+ | extend timestamp = TimeGenerated , IPCustomEntity = case(IPMatch == "SourceIP", SourceIp, IPMatch == "DestinationIP", DestinationIp, "None"), Host = Computer
+ ),
+ (Event
+ | where TimeGenerated >= ago(timeframe)
+ | where Source == "Microsoft-Windows-Sysmon"
+ | where EventID == 3
+ | extend EvData = parse_xml(EventData)
+ | extend EventDetail = EvData.DataItem.EventData.Data
+ | extend SourceIP = EventDetail.[9].["#text"], DestinationIP = EventDetail.[14].["#text"]
+ | where SourceIP in (IPList) or DestinationIP in (IPList) 
+ | extend IPMatch = case( SourceIP in (IPList), "SourceIP", DestinationIP in (IPList), "DestinationIP", "None") 
+ | extend timestamp = TimeGenerated, AccountCustomEntity = UserName, HostCustomEntity = Computer , IPCustomEntity = case(IPMatch == "SourceIP", SourceIP, IPMatch == "DestinationIP", DestinationIP, "None")
+ ), 
+ (WireData 
```

Comment:
Minor inconsistency with other sections where the DataType is on the same line as the bracket.  Either way is fine, but good to be consistent.

Reply:
Thanks!!
Changed now.